### PR TITLE
added modelling strategy and effect size and restrictions

### PR DESCRIPTION
**Pull request by Ninicorn**

### CODE REVIEW

File: docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst [hunk 1]
```diff
@@ -87,82 +87,93 @@ more than 280 kcal/day (> 90 g dose).LNS are nutrient dense, require no cooking
...
+    - Direct
+
 
 Baseline Coverage Data
 ++++++++++++++++++++++++
 
-.. todo::
-
-  Document known baseline coverage data, using the table below if appropriate
+No baseline coverage of SQ-LNS
```
Comment (aflaxman) [inline docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:146]: Does this mean "no data" or in the baseline/reference scenario, coverage is 0.0%?

File: docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst [hunk 2]
```diff
@@ -87,82 +87,93 @@ more than 280 kcal/day (> 90 g dose).LNS are nutrient dense, require no cooking
...
+We will apply the relative risk ratio as a relative rate ratio on the incidence of MAM from MILD (i2) starting from the age-start of the intervention starts (6 months). This is because we assume that the intervention does not affect the duration of disease and hence:
 
-  Add an overview of the Vivarium modeling section.
+| Prevalence_intervention 
+| = 0.82 x (prevalence_baseline_6mo) 
+| = 0.82 x (incidence_baseline_6mo) x Duration_baseline
 
 .. todo::
+  
+  Think about whether we want to apply the full reduction in incidence at 6 months or we have a gradual reduction in incidence, reaching full reduction at 18 months?
```
Comment (aflaxman) [inline docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:164]: Think about whether we need to apply a reduction to birth prevalence, as well, despite the illogic of an intervention at age 6 months affecting a risk exposure at 0 months.

**Review by aflaxman — APPROVED** [review]
Some temp file got inlcuded in this commit: `~$lns_memo_das2019.docx `

File: docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst (hunk 1 above)
Comment (Ninicorn) [reply docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:146]: The baseline coverage in 0.0%.

File: docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:164]: My thinking is that there is no reduction until 6 months. So 0-6 month olds stay the same as baseline.

File: docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst [hunk 3]
```diff
@@ -87,82 +87,93 @@ more than 280 kcal/day (> 90 g dose).LNS are nutrient dense, require no cooking
...
+  - Four studies included children aged six to 24 months (Dewey 2017; Olney 2018; Luby   2018; Null 2018) and 
+  - one study included children aged six to 36 months (Huybregts 2012). 
+  - Four included studies enrolled pregnant women and provided lipid-based nutrient supplements (LNS) plus complementary feeding during pregnancy and post-partum, followed by infant supplementation at six months of age (Adu-Afarwuah 2016; Ashorn 2015; Dewey 2017; Olney 2018). However, Dewey 2017 had an intervention arm in which only children were supplemented, hence we only used the data from that arm in the analysis in this review. The other studies provided LNS plus complementary feeding to children after six months of age. * The review conducted a sensitivity analysis by removing trials that also supplemented pregnant women with LNS in addition to children (Adu-Afarwuah 2016; Ashorn 2015), and found no significant difference in the outcome (Analysis 1.3; Analysis 1.4 in Das DAS_Cochrane_Review_2019).
```
Comment (NathanielBlairStahn) [inline docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:108]: What's with the asterisk before the last sentence?

> "... after six months of age. * The review conducted a sensitivity analysis..."

**Comment by NathanielBlairStahn** [comment]
> Some temp file got inlcuded in this commit: `~$lns_memo_das2019.docx`

Is the correct way to fix this to run `git rm ~$lns_memo_das2019.docx` and then commit the change?

See https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-a-git-repository/

Looks like you can also remove unwanted files directly on GitHub: https://github.blog/2013-07-04-deleting-files-on-github/

**Review by NathanielBlairStahn — APPROVED** [review]

File: docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst (hunk 2 above)
Comment (NathanielBlairStahn) [reply docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:164]: What are you thinking here, @aflaxman? Is the concern that somehow our prevalences might get thrown off again if we don't also alter birth prevalence?

Comment (aflaxman) [reply docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:164]: > What are you thinking here, @aflaxman? Is the concern that somehow our prevalences might get thrown off again if we don't also alter birth prevalence?

Exactly my concern.  But let's try it the logical way, and revisit if it does not look well-calibrated.

File: docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst (hunk 3 above)
Comment (Ninicorn) [reply docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:108]: oh...I just wanted to star that sentence to highlight it...

Comment (NathanielBlairStahn) [reply docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:108]: Oh ok, no problem.  I just wasn't sure if it was a .rst formatting error or something.

File: docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:164]: We don't expect prevalence to match GBD in this scenario. We are actively trying to decrease prevalence with a shorter duration (and incidence remain the same as baseline).

Comment (Ninicorn) [reply docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:164]: Hence prevalence of 0-6 month olds will be the same as baseline. Once the SQ-LNS intervention starts, the incidence of i2 decreases and we should see a decrease in prevalence among 6mo+.

Comment (Ninicorn) [reply docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:164]: We also expect 0% coverage of SQ-LNS at baseline.

Comment (Ninicorn) [reply docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst:164]: @aflaxman @NathanielBlairStahn

**Pull request merged**